plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1157,15 +1157,15 @@ private static IcebergColumnHandle createProjectedColumnHandle(IcebergColumnHand
     }
 
     @Override
-    public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTableHandle tableHandle, Constraint constraint)
+    public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTableHandle tableHandle)
     {
         if (!isStatisticsEnabled(session)) {
             return TableStatistics.empty();
         }
 
         IcebergTableHandle handle = (IcebergTableHandle) tableHandle;
         Table icebergTable = catalog.loadTable(session, handle.getSchemaTableName());
-        return TableStatisticsMaker.getTableStatistics(typeManager, constraint, handle, icebergTable);
+        return TableStatisticsMaker.getTableStatistics(typeManager, Constraint.alwaysTrue(), handle, icebergTable);
```

Comment:
Add TODO to clean this up.

This will let you simplify `io.trino.plugin.iceberg.BaseIcebergConnectorTest#getTableStatistics` and remove these tests

https://github.com/trinodb/trino/blob/ec3aa36ad17d04c72371de8b88b860bdb2667667/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java#L2009-L2025

Reply:
All cleaned up and I removed the test completely.